src/main/java/seedu/academyassist/model/AcademyAssist.java

```diff
@@ -109,10 +109,17 @@ public void sortPersonsByName() {
     }
 
     /**
-     * Sorts persons in this address book by class, in alphabetical order.
+     * Sorts persons in this address book by subject, in alphabetical order.
      */
-    public void sortPersonsByClass() {
-        persons.sortPersonsByClass();
+    public void sortPersonsBySubject() {
+        persons.sortPersonsBySubject();
+    }
+
+    /**
+     * Sorts persons in this address book by subject, in alphabetical order.
+     */
```

Comment:
I think there is a small mistake for the header comment here? is it supposed to be Sorts persons .... by year group, in ascending order?